src/coredump/coredump.c

```diff
@@ -48,6 +48,7 @@
 #include "tmpfile-util.h"
 #include "user-record.h"
 #include "user-util.h"
+#include "coredump-ratelimit.h"
```

Comment:
please keep this sorted

Reply:
Addressed the comment. Please check once.